dom.bs

```diff
@@ -1270,7 +1270,7 @@ for discussion).
 <ol>
  <li><p>If <var>event</var>'s <a>stop propagation flag</a> is set, then return.
 
- <li><p>Let <var>listeners</var> be a new <a>list</a>.
+ <li><p>Let <var>listeners</var> be a new <a for=/>list</a>.
```

Comment:
You mean we could put link defaults at a higher level than the source for WHATWG in Bikeshed somehow? That might be nice.